modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -1,39 +1,126 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-deploy.adoc
+// * logging/congig/cluster-logging-systemd
 
-[id="cluster-logging-collector-scaling_{context}"]
-= Scaling up systemd-journald 
+[id="cluster-logging-systemd-scaling_{context}"]
+= Configuring systemd-journald for cluster logging 
 
 As you scale up your project, the default logging environment might need some
 adjustments.
 
 For example, if you are missing logs, you might have to increase the rate limits for journald.
+You can adjust the number of messages to retain for a specified period of time to ensure that
+cluster logging does not use excessive resources without dropping logs. 
+
+You can also determine if you want the logs compressed, how long to retain logs, how or if the logs are stored,
+and other settings.
 
 .Procedure
 
-. Update to *systemd-219-22.el7.x86_64*.
+. Create a `journald.conf` file with the required settings:
++
+[source,yaml]
+----
+Compress=no <1>
+ForwardToConsole=yes <2>
+ForwardToSyslog=no <2>
+MaxRetentionSec=30s <3>
+RateLimitBurst=10000 <4>
+RateLimitInterval=1s <4>
+Storage=volatile <5>
+SyncIntervalSec=1s <6>
+SystemMaxUse=8g <7>
+SystemKeepFree=20% <8>
+SystemMaxFileSize10M <9>
+----
++
+<1> Specify whether you want logs compressed before they are written to the file system. 
+Specify `yes` to compress the message or `no` to not compress. Default is `yes`.
+<2> Configure whether to forward log messages. Defaults to `no` for each. Specify
+* `ForwardToConsole` to forward logs to the system console.
+* `ForwardToKsmg` to forward logs to the kernel log buffer.
+* `ForwardToSyslog` to forward to a syslog daemon.
+* `ForwardToWall` to forward messages to user's wall.
+<3> Specify the maximum time to store journal entries. Enter digits to specify seconds. Or 
+include a unit: "year", "month", "week", "day", "h" or "m". Enter 0 to disable. Default is `1month`. 
+<4> Congifure rate limiting. If, during the time interval defined by `RateLimitIntervalSec`, more logs than specified in `RateLimitBurst` 
+are received, all further messages within the interval are dropped until the interval is over. We recommend setting 
```

Comment:
We don't say "we". Suggest just "It is recommended.."

Reply:
It is awful... ;-)